nasa-gibs/worldview
# WV-2202: Disallow context menu outside map extent

## Description
- dont show context menu outside visible extent
- allow location marker in map wings

## How To Test

* Attempt to open context menu outside extent in all projections
* Place location marker in wings, confirm coordinates show correctly (e.g. longitude not greater than 180 or less than -180)
* Place location markers in polar projections
* Confirm coordinates in context menu match coordinates in location marker tooltip
* Confirm location markers cannot be placed outside visible extent
* Confirm location markers are placed properly on page refresh (url params)
